python_modules/dagster/dagster/_core/storage/runs/schema.py

```diff
@@ -148,6 +148,7 @@
 )
 
 db.Index("idx_run_tags", RunTagsTable.c.key, RunTagsTable.c.value, mysql_length=64)
+db.Index("idx_run_tags_run_idx", RunTagsTable.c.run_id, RunTagsTable.c.id, mysql_length=64)
```

Comment:
This should be accompanied with an alembic migration to add the index.

Reply:
Good find. I have added  migrations.

Reply:
I think this should be:
```
db.Index("idx_run_tags_run_idx", RunTagsTable.c.run_id, RunTagsTable.c.id, mysql_length={"run_id": 255})
```